src/stock-sources/stock-sources-items-table.component.tsx

```diff
@@ -72,6 +73,7 @@ function StockSourcesItems() {
 
   return (
     <div className={styles.tableOverride}>
+      <TabPanel>List of partners who provide stock to the facility.</TabPanel>
```

Comment:
we should try use translation here because eventual will ship this module to the community